name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/HTMLTableQuoteFeed.java

```diff
@@ -582,12 +591,12 @@ public static void main(String[] args) throws IOException
         PrintWriter writer = new PrintWriter(System.out); // NOSONAR
         for (String arg : args)
             if (arg.charAt(0) != '#')
-                doLoad(arg, writer);
+                new HTMLTableQuoteFeed().doLoad(arg, writer);
```

Comment:
I am not sure if this will really help you as the static test method always instantiates HTMLTableQuoteFeed.

Why not move the loop (in the main method) into the class as well?

Then each class can have a very simple local test method which is something like
```java
public static void main(String[] args) throws IOException
{
    new HTMLTableQuoteFeed().runWith(args);
}
```

However, I can merge the change nonetheless (once I am back at my laptop) - it is just a remark.

Reply:
It does help me as the only static test method is "main" now. So in my new class it just looks like this:
```
       for (String arg : args)
            if (arg.charAt(0) != '#')
                new CSQuoteFeed().doLoad(arg, writer);
```
doLoad is inherited from the HTML feed. So all fine so far. Of course this test framework could be improved a little bit if you want but as far as providing an URL as parameter and dumping the prices everything works fine with base class and derived class. I think if the rest is fine with you maybe just merge it please.